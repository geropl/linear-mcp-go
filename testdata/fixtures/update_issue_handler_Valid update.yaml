---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 560
        transfer_encoding: []
        trailer: {}
        host: api.linear.app
        remote_addr: ""
        request_uri: ""
        body: '{"query":"\n\t\tmutation UpdateIssue($id: String!, $input: IssueUpdateInput!) {\n\t\t\tissueUpdate(id: $id, input: $input) {\n\t\t\t\tsuccess\n\t\t\t\tissue {\n\t\t\t\t\tid\n\t\t\t\t\tidentifier\n\t\t\t\t\ttitle\n\t\t\t\t\tdescription\n\t\t\t\t\tpriority\n\t\t\t\t\turl\n\t\t\t\t\tcreatedAt\n\t\t\t\t\tupdatedAt\n\t\t\t\t\tstate {\n\t\t\t\t\t\tid\n\t\t\t\t\t\tname\n\t\t\t\t\t}\n\t\t\t\t\tteam {\n\t\t\t\t\t\tid\n\t\t\t\t\t\tname\n\t\t\t\t\t\tkey\n\t\t\t\t\t}\n\t\t\t\t}\n\t\t\t}\n\t\t}\n\t","variables":{"id":"TEST-10","input":{"title":"Updated Test Issue"}}}'
        form: {}
        headers:
            Content-Type:
                - application/json
            User-Agent:
                - linear-mcp-go/1.0.0
        url: https://api.linear.app/graphql
        method: POST
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: |
            {"data":{"issueUpdate":{"success":true,"issue":{"id":"1c2de93f-4321-4015-bfde-ee893ef7976f","identifier":"TEST-10","title":"Updated Test Issue","description":null,"priority":0,"url":"https://linear.app/linear-mcp-go-test/issue/TEST-10/updated-test-issue","createdAt":"2025-03-03T11:34:49.241Z","updatedAt":"2025-03-03T12:00:11.022Z","state":{"id":"42f7ad15-fca3-4d11-b349-0e3c1385c256","name":"Backlog"},"team":{"id":"234c5451-a839-4c8f-98d9-da00973f1060","name":"Test Team","key":"TEST"}}}}}
        headers:
            Alt-Svc:
                - h3=":443"; ma=86400
            Cache-Control:
                - no-store
            Cf-Cache-Status:
                - DYNAMIC
            Cf-Ray:
                - 91a8f8d44d59dcbe-FRA
            Content-Type:
                - application/json; charset=utf-8
            Date:
                - Mon, 03 Mar 2025 12:00:11 GMT
            Etag:
                - W/"1ed-Tr0BNHvSVQsgUsk1Wb6EJKfX7Po"
            Server:
                - cloudflare
            Vary:
                - Accept-Encoding
            Via:
                - 1.1 google
            X-Complexity:
                - "6"
            X-Ratelimit-Complexity-Limit:
                - "3000000"
            X-Ratelimit-Complexity-Remaining:
                - "2996924"
            X-Ratelimit-Complexity-Reset:
                - "1741006810967"
            X-Ratelimit-Requests-Limit:
                - "1500"
            X-Ratelimit-Requests-Remaining:
                - "1498"
            X-Ratelimit-Requests-Reset:
                - "1741006810967"
            X-Request-Id:
                - 91a8f8d46481dcbe-FRA
        status: 200 OK
        code: 200
        duration: 314.354649ms
