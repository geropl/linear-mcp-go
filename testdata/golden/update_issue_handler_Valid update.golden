err: ""
output: |-
    Updated Issue: TEST-10 (UUID: 1c2de93f-4321-4015-bfde-ee893ef7976f)
    URL: https://linear.app/linear-mcp-go-test/issue/TEST-10/updated-test-issue
